composer.json

```diff
@@ -62,23 +42,64 @@
         "typo3/cms-scheduler": "dev-main as 12.0",
         "typo3/cms-seo": "dev-main as 12.0",
         "typo3/cms-setup": "dev-main as 12.0",
+        "typo3/cms-styleguide": "dev-main",
         "typo3/cms-sys-note": "dev-main as 12.0",
         "typo3/cms-t3editor": "dev-main as 12.0",
         "typo3/cms-tstemplate": "dev-main as 12.0",
         "typo3/cms-viewpage": "dev-main as 12.0",
         "typo3/cms-workspaces": "dev-main as 12.0",
-        "ttn/tea": "^3.0",
-        "t3docs/examples": "dev-main",
-        "t3docs/site-package": "dev-main",
-        "typo3-documentation-team/speeddemo": "^11.4",
-        "typo3/cms-styleguide": "dev-main"
+        "typo3/coding-standards": "^0.5.5"
+    },
+    "repositories": {
+        "1a_git": {
+            "type": "vcs",
+            "url": "https://github.com/TYPO3-Documentation/tea.git"
+        },
+        "1c_git": {
+            "type": "vcs",
+            "url": "https://github.com/TYPO3-Documentation/blog_example.git"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "allow-plugins": {
+            "typo3/cms-composer-installers": true,
+            "typo3/class-alias-loader": true,
+            "ergebnis/composer-normalize": true
+        },
+        "bin-dir": ".Build/bin",
+        "sort-packages": true,
+        "vendor-dir": ".Build/vendor"
+    },
+    "extra": {
+        "typo3/cms": {
+            "extension-key": "t3docs-reference-coreapi",
```

Comment:
Do we need this? This is no extension, more like a project?

Reply:
TYPO3 now throws an error if a project with type "typo3-cms-*" does not have an extension. However the composer type "typo3-cms-documentation" is needed for the rendering. This has been in here since a while but the composer.json was reformated

Reply:
Okay, then leave it untouched ;-)